src/index.html

```diff
@@ -22,6 +22,367 @@
+
+          <div class="product_rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h3 class="product_rating_reviews">Reviews: 5</h3>
+          </div>
+
+          <div class="product_pricing">
+            <p class="product_pricing_line">Price:</p>
+            <p class="product_pricing_amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="product_buy"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="product">
+          <img
+            src="../src/images/imac.jpeg"
+            alt="iMac"
+            class="product_img"
+          />
+          <h2 class="product_name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <h3 class="product_code">Product code: 195434</h3>
+
+          <div class="product_rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h3 class="product_rating_reviews">Reviews: 5</h3>
+          </div>
+
+          <div class="product_pricing">
+            <p class="product_pricing_line">Price:</p>
+            <p class="product_pricing_amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="product_buy"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="product">
+          <img
+            src="../src/images/imac.jpeg"
+            alt="iMac"
+            class="product_img"
+          />
+          <h2 class="product_name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <h3 class="product_code">Product code: 195434</h3>
+
+          <div class="product_rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h3 class="product_rating_reviews">Reviews: 5</h3>
+          </div>
+
+          <div class="product_pricing">
+            <p class="product_pricing_line">Price:</p>
+            <p class="product_pricing_amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="product_buy"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="product">
+          <img
+            src="../src/images/imac.jpeg"
```

Comment:
The image path is incorrect. It should be 'images/imac.jpeg'. The path you've used is relative to the src directory, but the index.html file is also in the src directory.